### PR TITLE
Toolkit love: Templates

### PR DESCRIPTION
**Pull request by jhanstra**
Added information about the list-of-links and search-with-links widget templates. Also added a blurb about directives (which will be confirmed/changed when we decide how to document directives) and cleaned up some outdated things. 

Thoughts on other love the toolkit could use? I know there are a few issues open so I'll probably look at those next.

image
image

### CODE REVIEW

**Comment by vertein** [comment]
:+1:

File: my-uw/index.md [hunk 1]
```diff
@@ -245,79 +245,77 @@ slug: my-uw
...
-    </ul>
-    <a class="btn btn-default launch-app-button">See all</a>
-  </div>
+  <portlet-preference>
+      <name>widgetType</name>
+      <value>list-of-links</value>
+  </portlet-preference>
+  <portlet-preference>
+    <name>widgetConfig</name>
+    <value>{
```
Comment (apetro) [inline my-uw/index.md]: The `search-with-links` `widgetConfig` uses `<![CDATA[`; should this `widgetConfig`?

Comment (timlevett) [inline my-uw/index.md]: agreed

Comment (jhanstra) [inline my-uw/index.md]: yep, thanks

**Pull request merged**